controllers/volumereplication_controller.go

```diff
@@ -201,11 +203,18 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{Requeue: true}, nil
 	}
 
+	var msg string
 	instance.Status.LastCompletionTime = getCurrentTime()
 	if instance.Spec.ReplicationState == replicationv1alpha1.Resync {
 		err = r.updateReplicationStatus(instance, replicationv1alpha1.Replicating, "volume is marked for resyncing")
+		msg = "volume is marked for resyncing"
 	} else {
 		err = r.updateReplicationStatus(instance, replicationv1alpha1.Replicating, "volume is marked "+string(instance.Spec.ReplicationState))
+		msg = "volume is marked " + string(instance.Spec.ReplicationState)
+	}
+
+	if err == nil {
+		r.Log.Info(msg, "Name", instance.Name)
 	}
 
```

Comment:
I would suggest explicitly  returning the error after line 210 and line 213 in case `updateReplicationStatus` fails.  And return `nil` at the end.  Returning `err` at the end might be confusing.

Reply:
Done